examples/quest-2-step-options/qwiklabs.yaml

```diff
@@ -0,0 +1,89 @@
+entity_type: Quest
+schema_version: 1
+
+# Quest Attributes
+id: my-awesome-quest
+
+tags: [sample, life-changing, gcp]
+logo: ./quest-logo.png
+default_locale: en
+
+title:
+  locales:
+    en: GCP Survey Quest
+
+description:
+  locales:
+    en: Get a taste of what GCP has to offer.
+
+objectives:
+  locales:
+    en: |
+      <p>This course will teach people</p>
+      <ul><li>The power of frienship</li></ul>
+
+audience:
+  locales:
+    en: |
+      <p>This course is intended for people who</p>
+      <ul><li>Do not understand the power of frienship</li></ul>
+
+prerequisites:
+  locales:
+    en: <p>A firm understanding of interpersonal interactions is recommended but not required.</p>
+
+suggested_duration:
+  locales:
+    en: 1 day
+
+
+resources:
+  - type: video
+    ref_id: intro-video
+    locales:
+      en:
+        title: Welcome to GCP!
+        uri: https://youtu.be/dNW0B0HsvVs
+      es:
+        title: ¡Bienvenido a GCP!
+        uri: https://youtu.be/dNW0B0HsvVs
+  - type: video
+    ref_id: outro-video
+    locales:
+      en:
+        title: Wasn't that fun!
+        uri: https://youtu.be/dNW0B0HsvVs
+      es:
+        title: Eso fue divertido, ¿verdad?
+        uri: https://youtu.be/dNW0B0HsvVs
+
+steps:
+  - content: $intro-video
+  - content: intro-to-gcp
+  - choice:
+      prompt:
+        locales:
+          en: Learn about compute options
+      options:
+        - content: intro-to-appengine-python
```

Comment:
In `quest-2-step-options/whisperer.yaml`, steps nested under `options` have a `path` key:

```
  - choice:
      options:
        - path:
          - content: intro-to-appengine-python
```

But here you've omitted the `path` key (and the choices are simpler). Do you intend for `path` to be an optional nested attribute that is only required for complicated choices or was this just an accidental omission?

Reply:
The former: trying to make the syntax as reabable as possible.

I expect the qwiklabs.yaml will explicitly state a type here:

```yaml
- type: choice
  options:
    - type: path
    - steps:
      - type: Lab
        content: intro-to-appengine-python
```

Reply:
I was philosophically against Git Whisperer rewriting yaml structure in lab bundles but there aren't lab bundles and the specs already feel like they're diverging so I'll allow it.

Reply:
I *could* be convinced that qwiklabs.yaml doesn't require explicit types in the array, and we interpret it by what keys actual exist... or alternatively that we don't mind subjecting lab authors to the more verbose syntax.

As a pseudo-related aside, how do you feel about this situation?

```yaml
- type: choice
  options:
    - type: path
      steps:
        - type: Resource
          ref_id: my-appengine-intro-video
        - type: Lab
          content: intro-to-appengine-python
        - type: Lab
          content: intro-to-datastore-python
    - type: Lab
      content: massive-lab-that-is-equivalent-to-other-path
```

Reply:
Your original question was did I forget to add a path node. I think that confusion is reasonable.

- Do you think the mixture demonstrated above is ok or just begging for confused authors?
- Should we mandate that all options be the same type?
- Should we mandate that all options be a path even if they contain only one item?

Reply:
Adding `type` everywhere makes things much more verbose (and I already don't like how verbose my solution is). With that said, I prefer the syntax you used in `quest-3` (with nested paths)